recipe/symfony.php

```diff
@@ -22,17 +26,29 @@
 
 // Assets
 set('assets', ['web/css', 'web/images', 'web/js']);
-// Default true - BC for Symfony < 3.0
-set('dump_assets', true);
+
+// Requires non symfony-core package `kriswallsmith/assetic` to be installed
+set('dump_assets', false);
```

Comment:
@oanhnn 

The alternative to retain BC is to define `dump_assets` as `true` in symfony.php and `false` in symfony3.php but this causes inconsistent behavior between the two recipes, which I assert is more confusing than fixing the inconsistency. Thoughts?